### PR TITLE
READY: Fix d3 data updating

### LINKED ISSUE #296: [5] Inspector does not show right amount of neighbors
x

As shown, the correct amount of neighbors are not displayed of node 593. This has to do with the node being used to get the neighbor, it is not the updated node.

### PR DESCRIPTION
**Pull request by TimSpeelman**
Fixes #296

### CODE REVIEW

File: TriblerGUI/widgets/trustpage/js/radial_view/radial_node.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ function RadialNodes(svg, options) {
                 return node.public_key;
             });
 
+        update.each(function (e) {
+            console.log('new data', e.public_key, e.neighbors.length);
```
Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_node.js]: Remove debug functionality

File: TriblerGUI/widgets/trustpage/js/radial_view/radial_view.js [hunk 2]
```diff
@@ -39,6 +39,8 @@ function RadialView(svg, settings) {
         });
 
         self.nodes.bind("mouseover", function (targetNode) {
+            console.log('Entered',targetNode.public_key,'neighbors:',targetNode.neighbors.length);
```
Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_view.js]: Remove debug functionality

File: TriblerGUI/widgets/trustpage/js/radial_view/radial_link.js [hunk 3]
```diff
@@ -90,6 +90,12 @@ function RadialLinks(svg, options) {
      * @param updateSelection
      */
     self.update = function (updateSelection) {
+        // Pass updated data through to all child elements using it
+        updateSelection.each(function (newData) {
```
Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_link.js]: Why is this required?

Comment (TimSpeelman) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_link.js]: To make it future proof we make sure that all children of the link element also have the updated data.

**Pull request merged**